api/api.go

```diff
@@ -120,12 +125,36 @@ func (h *httpHandler) initRouter() {
 	}
 }
 
-func makeHandler(ctrl libnetwork.NetworkController, fct processor) http.HandlerFunc {
+const (
+	// only one Accept value for now
+	acceptRegex  = `^application/vnd\.libnetwork\.v\d\.\d\+json$`
```

Comment:
Not sure how important it is, but thought that adding the information wouldn't hurt.

Thanks for that link. I wasn't familiar with that approach; so if I understand correctly, a 'regular' `Accept: application/json` with that approach, will also work, and return the "latest" API?

Will more complex Accept-headers work (or does Go do the heavy lifting here?), e.g.
	
    Accept: application/xml; q=1, application/json; charset=utf-8; q=0.8